property_tests/arbitraries/candid/candid_value_and_meta_arb_generator.ts

```diff
@@ -34,8 +35,11 @@ export function CandidValueAndMetaArbGenerator<
                     imports,
                     valueLiteral
                 },
-                agentArgumentValue,
-                agentResponseValue
+                value: {
+                    agentArgumentValue,
+                    agentResponseValue,
+                    candidTypeObject: azleCandidTypeObject
```

Comment:
Deal with the naming here, seems like both should be named `runtimeCandidTypeObject`.